### ISSUE TITLE
Create a sub theming Starter Kit

### ISSUE THREAD

**Issue by stpaultim** [backdrop-contrib/tatsu#7]
Something like what ZEN has (Drupal 7).

See here for a list of current Backdrop CMS themes, with notes saying which ones have starter kits. 
https://docs.google.com/spreadsheets/d/15Cmp7jRsA_rpNXYPKsE1fNRGwjlu-m7rlKzIoirRJJI/edit#gid=0

**Comment by stpaultim** [comment]
I did a bunch of work on sub-theme starter kit today. I hope to get something merged yet this weekend. If I am able to do so, it would be great if someone was able to test the process for setting up a very simple sub-theme (not yet ready, but very soon).

**Comment by stpaultim** [comment]
Just added a sub-theming starter kit to 1.x-2.x-dev

I.x-2.x branch needs testing.

**Comment by izmeez** [comment]
Tried the starterkit for a subtheme with 1.x-2.x and have some mixed feelings. Partly because of issues with the color module which may be a totally separate issue. The other challenge with the startkit is to ensure files are in sync, e.g. skin.css in the tatsu theme and the starterkit are different. The main theme appears to have a few extra changes. This is the sort of extra housekeeping that could be an issue.

**Comment by Wylbur** [comment]
I just installed the 1.x-2.x branch, and instead of using the starter kit, I used the devel_subthemer module. This works great, with many features including color support and adding template files. 

Instead of replicating that work for this module, can this module point subtheme documentation to recommend using that module?  

Why reinvent the wheel?

**Comment by Wylbur** [comment]
If you agree, assign this to me and change the ticket name to Update documentation to recommend devel_subthemer module for subtheming.

**Comment by stpaultim** [comment]
Yes, I suppose that recommending and pointing at Devel Subthemer might be a good alternative. I'd like to try that myself. 

The key is to provide the documentation or point a themer at the best documentation to make it easy.

**Comment by izmeez** [comment]
I think I used the devel_subthemer module with better results. It was awhile ago so I'll have to try again with latest code.
